### DISCUSSION TITLE
Добавить возможность просмотра справочника внутренних телефонов

### DISCUSSION THREAD

**Discussion by EDautoAdmin** [mikopbx/Core#552] [Ideas (Идеи)]
Добрый день, коллеги.

Предлагаю идею предоставлении пользователям общего доступа к странице со списком пользователей.
ТЗ
1. Добавить справочник подразделений. Для начала можно без "веток". С одним уровнем.
2. Сделать у пользователя поле "Подразделение". Заполняется из справочника п.1
3. Сделать страничку, доступную для всех, на которой будет таблица типа в шапке, подразделение, в строках юзеры с телефонами и почтами.

Это самая простая реализация, но думаю, что она многим пригодится.

Далее эту тему можно развивать бесконечно. Например сделать возможность авторизации юзеров на странице и при нажатии на телефонный номер связывать юзера с тем номером, на который он нажал. Мы так делали на голом астере с помощью питона, но это требует авторизации. К стати можно и доменной по LDAP. :) ... но это уже так ... фантазии на тему второй версии модуля. Для начала исполнить по ТЗ. Проще некуда, на мой взгляд. И полезно.

**Comment by boffart** [comment]
См. модуль "Управление доступом в систему", он решает эту задачу.
